conn.go

```diff
@@ -666,6 +671,22 @@ func (cn *conn) simpleQuery(q string) (res *rows, err error) {
 	}
 }
 
+type noRows struct{}
+
+var emptyRows noRows
+
+var _ driver.Result = noRows{}
+
+var ErrNoRowsAffected = errors.New("no RowsAffected available after the empty statement")
```

Comment:
Have a better suggestion? I don't think it needs to be exported myself. Many other uncommon errors aren't.